docs/source/tutorials.pipelines.rst

```diff
@@ -0,0 +1,47 @@
+Pretrained Models and Pipelines
+===============================
+
+:py:mod:`torchaudio.pipelines` module provides access to pre-trained models and associated data processing pipelines to perform audio-related tasks.
+
+Automatic Speech Recognition (ASR)
```

Comment:
wondering if you think it could be nice to include these sections directly in the index (ex/ ASR, Forced Alignment, TTS, ...)? I think would be cool to highlight these capabilities/research areas directly on the sidebars, but several of them only consist of one tutorial so also makes sense to keep them in a section.